docs/guides/adding-custom-css.rst

```diff
@@ -77,3 +77,8 @@ you can include the theme's normal CSS using the CSS
     body {
         /* ... */
     }
+
+
+.. seealso::
+
+   You can also add custom classes to your html elements. See `Docutils Class <https://docutils.sourceforge.io/docs/ref/rst/directives.html#class>`_ and this related `Sphinx footnote... <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#id2>`_ for more information.
```

Comment:
Good catch! This was painfully difficult to find on the Sphinx docs, I will send a pull request there right now.

I think this information is too important to be in a `seealso` admonition! What about expanding it into its own subsection and adding a small example?

Reply:
Adding its own subsection sounds great. This is information that most people trying to add a custom CSS file will be looking for, so it makes sense to me to expand on it. Also, the information available today on the combined related docs is scarce and insufficient.

This `seealso` could become just a link to that new subsection.